index.js

```diff
@@ -50,6 +51,12 @@ function tryResolve(pkg, importer) {
 		return relative.resolve(pkg, importer);
 	} catch (err) {
 		if (err.code === 'MODULE_NOT_FOUND') return null;
+		if (err.code === 'ERR_PACKAGE_PATH_NOT_EXPORTED') {
+			if (!pkg_export_errors.has(pkg)) {
```

Comment:
No need to check has(). Just add it and Set will dedupe

Reply:
nice